src/Bridge/Doctrine/Orm/Extension/EagerLoadingExtension.php

```diff
@@ -35,6 +36,8 @@
  */
 final class EagerLoadingExtension implements QueryCollectionExtensionInterface, QueryItemExtensionInterface
 {
+    use ShouldEagerLoad;
```

Comment:
On another note, using a `Trait` suffix is still the Symfony CS, right?

Maybe `EagerLoadingTrait` \o/